apps/explorer/src/modules/tokens/components/TokenTableHolders.vue

```diff
@@ -178,7 +169,7 @@ export default class TokenTableHolders extends Vue {
     return MAX_ITEMS
   }
 
-  get loading() {
+  get loading(): boolean | undefined {
```

Comment:
@olgakup does this.$apollo return undefined? (Question)

Reply:
fixed in [ 5df127.. ](https://github.com/EthVM/EthVM/pull/729/commits/5df127701deb93c334add58d3c870dd6b9019f90)